### PR TITLE
Added Coderforces Scraping files to Codeforces_Scraper

### LINKED ISSUE #291: CodeForces Problem Scrapper
# Description
Develop a python script that should scrape the problem statement into a PDF File with a given problem code from the code forces website. The script should scrape the following items:

1. Problem Statement
2. Images (If any are mentioned in the problem statement)
3. Given Input and Output of the problem statement. 

## Language

- [x] Python

## Type of issue

- [x] Feature

### PR DESCRIPTION
**Pull request by iamakkkhil**
# Description

This python script will let you download Problem Statements from Codeforces and save them as a pdf file. The script uses Selenium Webdriver, Pillow and fpdf library. Selenium is used with Chrome Webdriver, so having Chrome browser is a requirement.

Fixes #291

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from selenium import webdriver  # Automated webdriver
+from PIL import Image
+from fpdf import FPDF  # For converting images to pdf
+
+
+def getproblem():
+    Pblm_id = input("Enter the Problem ID: ")
+    difficulty = input("Enter the difficulty level: ")
+    filename = input('Enter the file name to store Question: ') + '.pdf'
+
```
Comment (rahulraikwar00) [inline Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py:11]: @iamakkkhil add comments to make this code more readable?

File: Python/Coderforces_Problem_Scrapper/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pillow
+fpdf
+selenium
+
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/requirements.txt:4]: Remove the line

File: Python/Coderforces_Problem_Scrapper/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pillow
+fpdf
+selenium
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/requirements.txt:3]: Add package versions

File: Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import os
+from selenium import webdriver  # Automated webdriver
+from PIL import Image
+from fpdf import FPDF  # For converting images to pdf
+
+
+def getproblem():
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py:7]: Add a function docstring

File: Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    Pblm_id = input("Enter the Problem ID: ")
+    difficulty = input("Enter the difficulty level: ")
+    filename = input('Enter the file name to store Question: ') + '.pdf'
+
+    url = "https://codeforces.com/problemset/problem/" + Pblm_id + "/" + difficulty
+    path = 'image.png'
+    options = webdriver.ChromeOptions()
+
+    options.headless = True
+    driver = webdriver.Chrome("F:/chromedriver.exe", options=options)
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py:17]: Remove local references of your filepath

File: Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    pdf = FPDF(unit='pt', format=[WIDTH + 2 * MARGIN, HEIGHT + 2 * MARGIN])
+    pdf.add_page()  # Adding new page to the pdf
+    pdf.image(path, MARGIN, MARGIN)
+    pdf.output(filename, "F")
+
+    print(f'\nGreat Success!!! Check your directory for {filename} file!')
+
+
+getproblem()
+os.remove('image.png')
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py:35]: Use a main function to drive the code

File: Python/Coderforces_Problem_Scrapper/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+2.  Valid Difficulty Level.
+3. filename(without '.pdf'). The pdf will be created in the same folder.
+
+## PDF Output:
+![Imgur](https://i.imgur.com/GpSxCRZ.png)
+![Imgur](https://i.imgur.com/c5mCNWM.png)
+
+## Author
+[Akhil Bhalerao](https://github.com/iamakkkhil)
+
```
Comment (HarshCasper) [inline Python/Coderforces_Problem_Scrapper/README.md:36]: Remove this line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by iamakkkhil** [comment]
@HarshCasper updated files accordingly.

File: Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    pdf.add_page()  # Adding new page to the pdf
+    pdf.image(path, MARGIN, MARGIN)
+    pdf.output(filename, "F")
+
+    print(f'\nGreat Success!!! Check your directory for {filename} file!')
+
+
+if __name__ == "__main__":
+    getproblem()
+    os.remove('image.png')
```
Comment (vybhav72954) [inline Python/Coderforces_Problem_Scrapper/Codeforces_problem_scrapper.py:44]: Add an EOF file in the script.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
@HarshCasper The changes look good to me.

@iamakkkhil Please include some comments so that others can understand the workflow,

**Comment by iamakkkhil** [comment]
@vybhav72954 @HarshCasper Can you check this ?

**Review by vybhav72954 — APPROVED** [review]
@HarshCasper Good to go.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**